### PR TITLE
BOM-1524: Added python 3.8 in travis.yml

### PR DESCRIPTION
**Pull request by iamsobanjaved**
**JIRA:** [BOM-1524](https://openedx.atlassian.net/browse/BOM-1524)

### Changes:
- Added python 3.8 in travis.yml

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tox]
 skipsdist=True
-envlist = py35-django22-{static,pylint,tests,theme_static,check_keywords},py35-{isort,pycodestyle,extract_translations,dummy_translations,compile_translations, detect_changed_translations,validate_translations}
+envlist = {py35,py38}-django22-{static,pylint,tests,theme_static,check_keywords},py35-{isort,pycodestyle,extract_translations,dummy_translations,compile_translations, detect_changed_translations,validate_translations}
```
Comment (jmbowman) [inline tox.ini]: The second set of environments here should also have the option of running under 3.8.

File: .travis.yml [hunk 2]
```diff
@@ -1,53 +1,48 @@
...
-        DJANGO_ENV=django22
-        TESTNAME=test-python
-        TARGETS="requirements.js clean_static static validate_python"
-      after_success:
-        - pip install -U codecov
-        - docker exec ecommerce_testing /edx/app/ecommerce/ecommerce/.travis/run_coverage.sh
-        - codecov
+  - python: 3.5
+    env: DJANGO_ENV=django22 TESTNAME=quality-and-js TARGETS="requirements.js check_translations_up_to_date
+      validate_translations clean_static static quality validate_js check_keywords"
```
Comment (jmbowman) [inline .travis.yml]: Ugh, the script mangled the formatting pretty badly in this file.  Can we just go back to the original and manually add the new 3.8 entries?

File: tox.ini (hunk 1 above)
Comment (iamsobanjaved) [reply tox.ini]: Nice catch, thanks

File: .travis.yml (hunk 2 above)
Comment (iamsobanjaved) [reply .travis.yml]: Updated

**Review by jmbowman — APPROVED** [review]

**Pull request merged**